src/module/system/settings/homebrew/languages.ts

```diff
@@ -107,7 +107,10 @@ export class LanguagesManager {
         if (oldRarity === newRarity) return;
 
         const language = droppedEl.dataset.slug;
-        if (!objectHasKey(CONFIG.PF2E.languages, language) || language === "common") {
+        const isValidLanguage =
+            language &&
+            (objectHasKey(CONFIG.PF2E.languages, language) || this.menu.cache.languages.some((l) => l.id === language));
```

Comment:
This looks like an unrelated change

Reply:
Only marginally unrelated. There are two issues with updating homebrew languages.

1) Adding new ones
2) setting the rarity of freshly added ones before saved

This would be one entry in the changelog.

Reply:
I can try without this, if you'd rather I split this off, though I can't really see this being a second changelog entry.